tests/org.jboss.tools.fuse.ui.bot.test/src/org/jboss/tools/fuse/ui/bot/test/RegressionTest.java

```diff
@@ -494,4 +501,52 @@ public void issue_2051() {
 			fail("Disabled project type is selected - see https://issues.jboss.org/browse/FUSETOOLS-2051");
 		}
 	}
+
+	/**
+	 * <p>
+	 * Local Launch doesn't work all times
+	 * </p>
+	 * <b>Link:</b>
+	 * <a href="https://issues.jboss.org/browse/FUSETOOLS-2062">https://issues.jboss.org/browse/FUSETOOLS-2062</a>
+	 */
+	@Test
+	public void issue_2062() {
+
+		NewFuseIntegrationProjectWizard wiz = new NewFuseIntegrationProjectWizard();
```

Comment:
try to use ProjectFactory.newProject("test-empty").version(CAMEL_LATEST).type(ProjectType.SPRING).create();